src/peer/node.js

```diff
@@ -0,0 +1,82 @@
+const libp2p = require('libp2p-ipfs')
```

Comment:
This appears to mostly be a bundle of other js-libp2p libs, only really adding the `Node` class. I wonder if it's the right thing to use -- let's ask IPFS.